RLTest/env.py

```diff
@@ -132,20 +133,20 @@ def compareEnvs(self, env):
                 return False
         return True
 
-    def __init__(self, testName=None, testDescription=None, module=None, moduleArgs=None, env=None, useSlaves=None, shardsCount=None,
-                 useAof=None):
+    def __init__(self, testName=None, testDescription=None, module=None, moduleArgs=None, env=None, useSlaves=None, shardsCount=None, useAof=None, decode_responses=False):
```

Comment:
I do not think we should allow the test to override decode_responses, also notice the convention (should be decodeResponses)

Reply:
For the most part I stuck with the convention from the redis module, which is where this parameter is ultimately passed.  I figure python developers are likely to be familiar with the redis module, so best to stick with that convention so its clear this parameter is passed through. For the default setting I have used the RLTest camel-case convention.